platforms/Windows/bundle/installer.wxs

```diff
@@ -59,6 +59,25 @@
     -->
 
     <Chain>
+      <?if $(VCREDIST_INSTALLER) != ""?>
+        <ExePackage
```

Comment:
I recommend using `BundlePackage` instead; it handles detection and versioning for you. Right now, the redist would be installed every time, which would lead to failure when somebody has a later version already installed.

Reply:
Great to know, thanks a lot for the tip! I will investigate

Reply:
I migrated to `<BundlePackage>` (and using aka.ms as the download url). Thanks!